addons/storyshots/storyshots-core/src/frameworks/svelte/loader.js

```diff
@@ -0,0 +1,32 @@
+import global from 'global';
```

Comment:
I don't see any committed snapshots, does this addon really work?

Reply:
Not yet, I had to stop and sleep yesterday, got a bit late 😸

Reply:
This should now be generating snapshots.